src/MessageProperty/Conventions.cs

```diff
@@ -25,6 +25,11 @@ public bool IsEncryptedProperty(PropertyInfo property)
             }
         }
 
-        internal Func<PropertyInfo, bool> IsEncryptedPropertyAction = p => typeof(WireEncryptedString).IsAssignableFrom(p.PropertyType);
+        /// <summary>
+        /// The convention.
+        /// </summary>
```

Comment:
might be that we need to disable a warning in case we need to target core 6.2 (which we don't at the moment and I'm not sure we ever need to)